### PR TITLE
fix: (Core) fix the accessibility bug in file uploader component.

### PR DESCRIPTION
**Pull request by manu-kr**
#### Please provide a link to the associated issue.
fixes: https://github.com/SAP/fundamental-ngx/issues/3497
fixes: https://github.com/SAP/fundamental-ngx/issues/3498

#### Please provide a brief summary of this pull request.
Placeholder text and tooltip information should be conveyed to the user.
Complete name of the UI or text on the button should be read out when accessed.

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [NA] update `README.md`
- [NA] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: apps/docs/src/app/core/component-docs/file-uploader/examples/file-uploader-drag-disabled-example/file-uploader-drag-disabled-example.component.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
     ariaLabelledBy="Choose file"
     placeholder="Choose File to upload"
     buttonLabel="Drag a .png file on me!"
-    buttonAriaLabel="browse file"
+    buttonAriaLabel="Drag a .png file on me!"
```
Comment (InnaAtanasova) [inline apps/docs/src/app/core/component-docs/file-uploader/examples/file-uploader-drag-disabled-example/file-uploader-drag-disabled-example.component.html:6]: "Drag a .png file OVER me!"

**Comment by InnaAtanasova** [comment]
Maybe I am missing something but why is this PR starting with `bug`?

**Review by droshev — CHANGES_REQUESTED** [review]
Can you update the title following the semantic versioning - `fix: `

File: apps/docs/src/app/core/component-docs/file-uploader/examples/file-uploader-drag-disabled-example/file-uploader-drag-disabled-example.component.html (hunk 1 above)
Comment (manu-kr) [reply apps/docs/src/app/core/component-docs/file-uploader/examples/file-uploader-drag-disabled-example/file-uploader-drag-disabled-example.component.html]: Modified.

**Review by droshev — CHANGES_REQUESTED** [review]
The PR title doesn't help us understand what it is fixing

**Comment by manu-kr** [comment]
> The PR title doesn't help us understand what it is fixing

>Modified PR title.

**Review by droshev — CHANGES_REQUESTED** [review]
I am not sure what value brings to our users this title:
`fix: (Core) fixes defect #3497 and #3498....
`
Modified.

**Review by InnaAtanasova — APPROVED** [review]

**Review by droshev — APPROVED** [review]

**Pull request merged**